packages/neuron-wallet/src/channel/index.ts

```diff
@@ -248,6 +249,26 @@ const listenToChannel = () => {
       })
     }, 1000)
   })
+
+  /**
+   * @name setNetwork
+   */
+  ipcMain.on(Channel.SetNetwork, (e: Electron.Event, network: { name: string; remote: string }) => {
+    // TODO:
```

Comment:
I suggest we persist all networks and current network to `electron-store` as JSON file. Note do not save to `keys` (wallets) folders.